webapp/build.gradle

```diff
@@ -39,6 +39,8 @@ dependencies {
 
     compile 'org.apache.commons:commons-csv:+'
 
+    compile 'com.google.code.gson:gson'
```

Comment:
I'd prefer using the existing jackson ObjectMapper for json serialization/deserialization.